src/i18n/lang/fr.json

```diff
@@ -101,7 +101,8 @@
     "addVersionButtonText": "Ajouter une version",
     "createModelButtonText": "Définir comme modèle",
     "manageAccessButtonText": "Gérer les accès",
-    "openViewerButtonText": "Voir",
+    "openViewerButtonText": "Ouvrir",
```

Comment:
@NicolasRichel does changing French translation work in POEditor?

Reply:
Yes it works.

Reply:
It should also mark translations in other languages as "**fuzzy**" so that we know they need to be updated. ;-)